modules/lightning_features/lightning_preview/lightning_preview.install

```diff
@@ -51,3 +51,10 @@ function lightning_preview_update_8001() {
   $display['display_options']['filters']['current_workspace'] = unserialize('a:15:{s:2:"id";s:17:"current_workspace";s:5:"table";s:15:"node_field_data";s:5:"field";s:17:"current_workspace";s:12:"relationship";s:4:"none";s:10:"group_type";s:5:"group";s:11:"admin_label";s:0:"";s:8:"operator";s:1:"=";s:5:"value";s:0:"";s:5:"group";i:1;s:7:"exposed";b:0;s:6:"expose";a:10:{s:11:"operator_id";s:0:"";s:5:"label";s:0:"";s:11:"description";s:0:"";s:12:"use_operator";b:0;s:8:"operator";s:0:"";s:10:"identifier";s:0:"";s:8:"required";b:0;s:8:"remember";b:0;s:8:"multiple";b:0;s:14:"remember_roles";a:1:{s:13:"authenticated";s:13:"authenticated";}}s:10:"is_grouped";b:0;s:10:"group_info";a:10:{s:5:"label";s:0:"";s:11:"description";s:0:"";s:10:"identifier";s:0:"";s:8:"optional";b:1;s:6:"widget";s:6:"select";s:8:"multiple";b:0;s:8:"remember";b:0;s:13:"default_group";s:3:"All";s:22:"default_group_multiple";a:0:{}s:11:"group_items";a:0:{}}s:11:"entity_type";s:4:"node";s:9:"plugin_id";s:17:"current_workspace";}');
   $view->save();
 }
+
+/**
+ * Clears the entity type definition cache.
+ */
+function lightning_preview_update_8002() {
```

Comment:
We need these because when we override the entity forms and handlers, we're changing the entity type definitions and need to force those changes to be picked up. Normally this would happen during a cache clear caused by a database update, if there was no database update, the changes would not be picked up and things would fatal left and right. So this prevents that from happening.